storage/multicdnstorage.py

```diff
@@ -0,0 +1,195 @@
+import logging
+from typing import Dict
+
+from storage.basestorage import BaseStorageV2, InvalidConfigurationException
+from util.ipresolver import GEOIP_CONTINENT_CODES
+
+logger = logging.getLogger(__name__)
+
+from storage.storagecontext import StorageContext
+from storage.storagedriverclasses import STORAGE_DRIVER_CLASSES
```

Comment:
I don't think you need the full list of storage drivers here, just the CDN specific ones, assuming you subclass the S3Storage class.

Reply:
added classes just for the multiCDN provider